jaxtyping/_import_hook.py

```diff
@@ -88,14 +92,71 @@ def _dot_lookup(*elements):
     return out
 
 
-def _str_lookup(string):
-    module = ast.parse(string)
-    (expr,) = module.body
-    return expr.value
+class _Typechecker:
+    lookup = {}
+
+    def __init__(self, typechecker):
+        self.typechecker = typechecker
```

Comment:
I don't think this is used anywhere, i.e. you can avoid saving it as an attribute.

Reply:
Yep! Thanks